hail/src/main/scala/is/hail/types/physical/stypes/interfaces/SNDArray.scala

```diff
@@ -244,6 +245,65 @@ object SNDArray {
     }
   }
 
+  def copy(cb: EmitCodeBuilder, _X: SNDArrayCode, _Y: SNDArrayCode): Unit = {
```

Comment:
I feel like something about the name of this function should suggest that it is only for copying vectors.

Reply:
Good point. I was just copying blas/lapack names (dropping the first element type letter), but I can make this more descriptive.